cmd/example-provider/main.go

```diff
@@ -42,6 +45,19 @@ func main() {
 		klog.Fatal(err)
 	}
 
+	if err = (&controllers.ClusterReconciler{
```

Comment:
`manager.New` is just `ctrl.NewManager`, `config.GetConfigOrDie` is `ctrl.GetConfigOrDie`, and `signals.SetupSignalHandler` is `ctrl.SetupSignalHandler`, if you want to prune some imports

Reply:
Nice, I'll rework it, thanks for the heads up